<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArithmeticOpWizardPageForm</class>
 <widget class="QWizardPage" name="ArithmeticOpWizardPageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arithmetic Operations</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_26">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Operation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_23">
      <item row="0" column="0">
       <widget class="QTabWidget" name="m_tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::West</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Default</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_25">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_24">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_8">
              <property name="title">
               <string>Layers</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_12">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_5">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>A:</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_3">
                     <property name="text">
                      <string>B:</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="1">
                  <layout class="QGridLayout" name="gridLayout_2">
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_layerAComboBox">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QComboBox" name="m_layerBComboBox">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="2">
                  <layout class="QGridLayout" name="gridLayout_3">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_2">
                     <property name="text">
                      <string>Band:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Band:</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="3">
                  <layout class="QGridLayout" name="gridLayout_4">
                   <item row="0" column="0">
                    <widget class="QComboBox" name="m_bandAComboBox">
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0">
                    <widget class="QComboBox" name="m_bandBComboBox">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>50</width>
                       <height>16777215</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Type</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_7">
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="m_op1RadioButton">
                   <property name="text">
                    <string>Gain * A + Offset</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="m_op2RadioButton">
                   <property name="text">
                    <string>Gain * ( A + B ) + Offset</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QRadioButton" name="m_op3RadioButton">
                   <property name="text">
                    <string>Gain * ( A - B ) + Offset</string>
                   </property>
                  </widget>
                 </item>
                 <item row="3" column="0">
                  <widget class="QRadioButton" name="m_op4RadioButton">
                   <property name="text">
                    <string>Gain * ( A * B ) + Offset</string>
                   </property>
                  </widget>
                 </item>
                 <item row="4" column="0">
                  <widget class="QRadioButton" name="m_op5RadioButton">
                   <property name="text">
                    <string>Gain * ( A / B ) + Offset</string>
                   </property>
                  </widget>
                 </item>
                 <item row="5" column="0">
                  <widget class="QRadioButton" name="m_op6RadioButton">
                   <property name="text">
                    <string>( Gain * ( A - B ) / ( A + B )) + Offset</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Params</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_8">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_5">
                     <property name="text">
                      <string>Gain</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="m_gainLineEdit">
                     <property name="text">
                      <string>1.0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="1">
                  <layout class="QGridLayout" name="gridLayout_9">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_6">
                     <property name="text">
                      <string>Offset</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QLineEdit" name="m_offsetLineEdit">
                     <property name="text">
                      <string>0.0</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item row="0" column="2">
                  <widget class="QCheckBox" name="m_normalizeCheckBox">
                   <property name="text">
                    <string>Normalize</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>User Defined</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_22">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_13">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_4">
                <property name="title">
                 <string>Operator</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_15">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_16">
                   <item row="0" column="0">
                    <widget class="QToolButton" name="m_leftBracketToolButton">
                     <property name="text">
                      <string>(</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QToolButton" name="m_rightBracketToolButton">
                     <property name="text">
                      <string>)</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QToolButton" name="m_plusToolButton">
                     <property name="text">
                      <string>+</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QToolButton" name="m_minusToolButton">
                     <property name="text">
                      <string>-</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="4">
                    <widget class="QToolButton" name="m_multiToolButton">
                     <property name="text">
                      <string>*</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="5">
                    <widget class="QToolButton" name="m_divToolButton">
                     <property name="text">
                      <string>/</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QGroupBox" name="groupBox_5">
                <property name="title">
                 <string>Value</string>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <layout class="QGridLayout" name="gridLayout_17">
                 <item row="0" column="0">
                  <layout class="QGridLayout" name="gridLayout_18">
                   <item row="0" column="0">
                    <widget class="QLineEdit" name="m_valueLineEdit">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="1">
                    <widget class="QToolButton" name="m_addValueOpToolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string>Raster / Band / ID</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_19">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_20">
                 <item row="0" column="0">
                  <widget class="QComboBox" name="m_layerComboBox">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="m_bandComboBox">
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QToolButton" name="m_addRasterOpToolButton">
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QComboBox" name="m_rasterIdComboBox">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>50</width>
                     <height>16777215</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string>Expression</string>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
              <layout class="QGridLayout" name="gridLayout_28">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_27">
                 <item row="0" column="0">
                  <widget class="QLineEdit" name="m_expressionLineEdit">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <layout class="QGridLayout" name="gridLayout_21">
                   <item row="0" column="0">
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="0" column="1">
                    <widget class="QToolButton" name="m_undoToolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="2">
                    <widget class="QToolButton" name="m_redoToolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="3">
                    <widget class="QToolButton" name="m_clearToolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                    </widget>
                   </item>
                   <item row="0" column="4">
                    <widget class="QToolButton" name="m_validateToolButton">
                     <property name="text">
                      <string>...</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_layerAComboBox</tabstop>
  <tabstop>m_bandAComboBox</tabstop>
  <tabstop>m_layerBComboBox</tabstop>
  <tabstop>m_bandBComboBox</tabstop>
  <tabstop>m_op1RadioButton</tabstop>
  <tabstop>m_op2RadioButton</tabstop>
  <tabstop>m_op3RadioButton</tabstop>
  <tabstop>m_gainLineEdit</tabstop>
  <tabstop>m_offsetLineEdit</tabstop>
  <tabstop>m_normalizeCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_op1RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_layerBComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op1RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_bandBComboBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op2RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_layerBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op2RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_bandBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op3RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_bandBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>397</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op4RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_layerBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op4RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_bandBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>133</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op5RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_layerBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op5RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_bandBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op6RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_layerBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op6RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_bandBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>246</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_op3RadioButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>m_layerBComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
